packages/now-client/src/create-deployment.ts

```diff
@@ -114,10 +114,24 @@ export default function buildCreateDeployment(version: number) {
     let configPath: string | undefined;
     if (!nowConfig) {
```

Comment:
`nowConfig` is always an object passed from the CLI so this code is never executed